gettsim/parameters/wohngeld.yaml

```
@@ -372,6 +372,57 @@ koeffizienten_berechnungsformel:
         a: -0.14
         b: 0.000101
         c: 0.0000513
+    2022-01-01:
+      note: Eine neuartige Verordnung regelt die Forschreibung des Wohngeldes.
```

```suggestion
      note: Eine neuartige Verordnung regelt die Fortschreibung des Wohngeldes.
```

Not fully sure here, though. Is this really about the process in the future? Why do we care that it is of a new type?

---

Because you have to look now at another place for the actual parameters, as indicated in the note at the top. To make things worse, the intial law is not changed, but just outdated.

---

Then we should say so explicitly! :smile:

---

The "new" part is that these parameters are now found in this WoGV and not in WoGG anymore. Just wanted to highlight that, as it can be confusing that WoGG still has (obsolete) parameters in the appendix. I'm fine with dropping this comment as we now have this in the description key.

---

then let's drop it